src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DropColumns_SupportColumnNamesAsIdentifiers.txt

```diff
@@ -0,0 +1,28 @@
+﻿#SETUP: SupportColumnNamesAsIdentifiers
+
+>> DropColumns(Table({a:1,b:2},{a:3,b:4}), b)
+Table({a:1},{a:3})
+
+>> DropColumns(Table({a:1,b:2},{a:3,b:4}), b, a)
+Table({},{})
+
+>> DropColumns(Table({a:1,b:2},{a:3,b:4}), a, b)
+Table({},{})
+
+>> DropColumns(DropColumns(Table({a:1,b:2},{a:3,b:4}), a), b)
```

Comment:
>DropColumns(DropColumns(Table({a:1,b:2},{a:3,b:4}), a), b) [](http://example.com/codeflow?start=3&length=58)

Can you add a case for DropColumns(AddColumns(...?

Reply:
>> DropColumns(AddColumns(Table({a:1,b:2},{a:3,b:4}), c, a * b), a)
Table({b:2,c:2},{b:4,c:12})

>> DropColumns(AddColumns(Table({a:1,b:2},{a:3,b:4}), c, a * b), b)
Table({a:1,c:2},{a:3,c:12})

>> DropColumns(AddColumns(Table({a:1,b:2},{a:3,b:4}), c, a * b), c)
Table({a:1,b:2},{a:3,b:4})

>> AddColumns(DropColumns(Table({a:1,b:2},{a:3,b:4}), a), a, b * b * b)
Table({a:8,b:2},{a:64,b:4})